lib/scheduler.registry.ts

```diff
@@ -33,14 +34,56 @@ export class SchedulerRegistry {
   }
 
   addCron(name: string, job: CronJob) {
```

Comment:
What about changing `addCron` to `addCronJob` to stay consistent?

Reply:
Makes sense.  Will update.